hal/src/argon/network/network.cpp

```diff
@@ -173,6 +169,28 @@ int if_init_platform(void*) {
     return 0;
 }
 
+#if HAL_PLATFORM_IF_INIT_POSTPONE
+int if_init_platform_postpone(void*) {
+    if (en2) {
+        uint8_t dummy;
+        if (!if_get_index(en2->interface(), &dummy)) {
+            auto wizNetif = reinterpret_cast<WizNetif*>(en2);
+            WizNetifConfigData wizNetifConfigData;
+            wizNetifConfigData.size = sizeof(WizNetifConfigData);
+            wizNetifConfigData.version = WIZNETIF_CONFIG_DATA_VERSION;
+            WizNetifConfig::instance()->getConfigData(&wizNetifConfigData);
+            if (wizNetif->init(wizNetifConfigData) == 0) {
+                return 0;
+            }
+            LOG(INFO, "Remove Ethernet interface");
```

Comment:
Should this be more like `LOG(ERROR, "Error Initializing Ethernet");`

I won't duplicate my comment for all of the rest of them ;-)

Reply:
Yeah, might be better to make it something like `not detected`